### ISSUE TITLE
🐛 Bug Report: Malformed response when creating email session in Flutter

### ISSUE THREAD

**Issue by AlbertoBonfiglio** [appwrite/appwrite#6250]
### 👟 Reproduction steps

1. Set up a project with a custom domain and email authorization enabled in Appwrite 1.4.1.
2. create a sample flutter app with the latest appwrite flutter  library
3. create a user 
4. attempt to create  the same user 

### 👍 Expected behavior

It should throw a 400 exception. The AppwriteException should have a valid code, message, response, and type.

### 👎 Actual Behavior

It correctly throws an Appwrite exception but the response is malformed. It returns code, response and type as null, and message as a long winded html page.

![Screenshot 2023-09-05 105246](https://github.com/appwrite/appwrite/assets/15167415/6868adce-4f84-4e49-8676-7f7b8d3fccef)

When looking at the web console, i can see the correct error is returned in the post but not in the flutter exception

![Screenshot 2023-09-05 105246](https://github.com/appwrite/appwrite/assets/15167415/24e5edf9-5e75-4fd1-bd37-2e8dd739d93c)

 

### 🎲 Appwrite version

Version 1.4.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Standard docker deployment of 1.4.1 in linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by ItzNotABug** [comment]
I believe there were issues on `1.4` & `1.4.1`, can you try migrating to `1.4.2` & see if that helps?

**Comment by stnguyen90** [comment]
Also, did you include the `/v1` for your endpoint?

**Comment by stnguyen90** [comment]
@AlbertoBonfiglio, are you still facing this problem? FYI, I'll need to close this due to inactivity soon.

**Comment by AlbertoBonfiglio** [comment]
> Also, did you include the `/v1` for your endpoint?

Yes I did. I'll do a migration to the latest  version this week and report back.

**Comment by AlbertoBonfiglio** [comment]
As of 1.4.5 it works correctly.  I did not test versions in between.
